### ISSUE TITLE
No translation when using Argon HUD

### ISSUE THREAD

**Issue by Lioheart** [Saibot393/enhancedcombathud-pf2e#31]
When using Argon HUD and system translation at the same time, translations of some actions displayed in Argon HUD do not appear.

![obraz](https://github.com/Saibot393/enhancedcombathud-pf2e/assets/39154364/10d83659-c71c-4bd1-9b7f-6222cc7c17ac)

When the module is disabled, the Hide action is translated correctly. This applies to system translations in all languages.

**Comment by Lioheart** [comment]
Thanks, I'd appreciate it

**Comment by Lioheart** [comment]
The Russian translators tried to solve this by adding the following code

```
Hooks.on('babele.ready', () => {
  for (const pack of game.packs.values()) {
    if (pack.size > 0) {
      for (const k of pack.keys()) {
        pack.delete(k);
      }
    }
   }
 })
```

**Comment by Saibot393** [comment]
Thank you, i will play around with the code, seems to be a good start

**Comment by Cuingamehtar** [comment]
To be more specific, I used that to sort-of solve the general Babele issue where the item is cached in the compendium before it is translated by Babele (in case it's linked in chat history, for example).
It doesn't look like `game.pf2e.actions.get(action)` actually creates such cache, so I'm not sure that'll work.

**Comment by Saibot393** [comment]
For the in panel actions i am actually using the action items. From my testing the ```game.pf2e.actions.get(action)``` do actually not seem to havy any problem with the translations so i might need to change the panel actions to those

**Comment by Cuingamehtar** [comment]
It might be better to do that, I think, as those get their names and content from core i18n subsystem, and don't depend on the behavior of third party modules.

**Comment by Saibot393** [comment]
The bug should be solved with v1.8.5, actions are now loaded from ```game.pf2e.actions````instead of the item compendiums, this will also make it easier to implement "on the fly" changes for the actions in the future
